Symfony/CS/Fixer/Contrib/PhpdocToCommentFixer.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Ceeram <ceeram@cakephp.org>
+ */
+class PhpdocToCommentFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        foreach ($tokens->findGivenKind(T_DOC_COMMENT) as $index => $token) {
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
+
+            // skip if there is no next token or if next token is block end `}`
+            if (null === $nextIndex || $tokens[$nextIndex]->equals('}')) {
+                continue;
+            }
+
+            if ($this->isStructuralElement($tokens[$nextIndex])) {
+                continue;
+            }
+
+            if ($tokens[$nextIndex]->isGivenkind(T_FOREACH) && $this->isValidForeach($tokens, $index)) {
+                continue;
+            }
+
+            $token->setContent('/*'.ltrim($token->getContent(), '/*'));
```

Comment:
this is wrong. The token will still be marked as a T_DOC_COMMENT, which might break things (unlikely here because each iteration only looks at later parts of the collection and the collection is dropped after that, but totally sure in the 2.0 version where the Tokens collection is not recreated for each fixer).

You should be using ``$token->override`` instead